pkg/exporter/ipfix.go

```diff
@@ -294,31 +294,31 @@ func setIEValue(record *model.Record, ieValPtr *entities.InfoElementWithValue) {
 	ieVal := *ieValPtr
 	switch ieVal.GetName() {
 	case "ethernetType":
-		ieVal.SetUnsigned16Value(record.Id.EthProtocol)
+		ieVal.SetUnsigned16Value(record.Metrics.EthProtocol)
 	case "flowDirection":
-		ieVal.SetUnsigned8Value(record.Id.Direction)
+		ieVal.SetUnsigned8Value(record.ID.Direction)
 	case "sourceMacAddress":
-		ieVal.SetMacAddressValue(record.Id.SrcMac[:])
+		ieVal.SetMacAddressValue(record.Metrics.SrcMac[:])
 	case "destinationMacAddress":
-		ieVal.SetMacAddressValue(record.Id.DstMac[:])
+		ieVal.SetMacAddressValue(record.Metrics.DstMac[:])
 	case "sourceIPv4Address":
-		setIPv4Address(ieValPtr, model.IP(record.Id.SrcIp).To4())
+		setIPv4Address(ieValPtr, model.IP(record.ID.SrcIp).To4())
 	case "destinationIPv4Address":
-		setIPv4Address(ieValPtr, model.IP(record.Id.DstIp).To4())
+		setIPv4Address(ieValPtr, model.IP(record.ID.DstIp).To4())
 	case "sourceIPv6Address":
-		ieVal.SetIPAddressValue(record.Id.SrcIp[:])
+		ieVal.SetIPAddressValue(record.ID.SrcIp[:])
 	case "destinationIPv6Address":
-		ieVal.SetIPAddressValue(record.Id.DstIp[:])
+		ieVal.SetIPAddressValue(record.ID.DstIp[:])
 	case "protocolIdentifier", "nextHeaderIPv6":
-		ieVal.SetUnsigned8Value(record.Id.TransportProtocol)
+		ieVal.SetUnsigned8Value(record.ID.TransportProtocol)
 	case "sourceTransportPort":
-		ieVal.SetUnsigned16Value(record.Id.SrcPort)
+		ieVal.SetUnsigned16Value(record.ID.SrcPort)
 	case "destinationTransportPort":
-		ieVal.SetUnsigned16Value(record.Id.DstPort)
+		ieVal.SetUnsigned16Value(record.ID.DstPort)
 	case "icmpTypeIPv4", "icmpTypeIPv6":
-		ieVal.SetUnsigned8Value(record.Id.IcmpType)
+		ieVal.SetUnsigned8Value(record.ID.IcmpType)
 	case "icmpCodeIPv4", "icmpCodeIPv6":
-		ieVal.SetUnsigned8Value(record.Id.IcmpCode)
+		ieVal.SetUnsigned8Value(record.ID.IcmpCode)
```

Comment:
this change from `Id` to `ID` generated tons of unnecessary changes here in many other places

Reply:
yeah but it's the naming convention & makes golint happier ...

Reply:
might make linter happier but screw up all inflightes PRs :(